Berksfile

```diff
@@ -1,5 +1,7 @@
 source 'https://supermarket.chef.io'
 
+cookbook 'poise-python', '~> 1.5.1'
```

Comment:
The version dependency can be declared in the metadata file so tools other than Berkshelf can see the version constraint. I'll comment over on the line you can update!